kubernetes/argocd/stacks/plausible/postgres.yml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: postgres-pvc
+  namespace: plausible
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: mayastor-3
```

Comment:
We may want to consider looking into why `nfs-csi` wasn't working. `mayastor-3` doesn't support the `RWX` access mode which enables rolling deployments.